Selenium_Basics/Selenium_Basics/Selenium_Basics_2/EpamPageInteractionTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+using System.Collections.Specialized;
+
+namespace Selenium_Basics.Selenium_Basics_2
+{
+    public class EpamPageInteractionTests
+    {
+        private Actions _action;
+        private IWebDriver _driver;
+        private const string _searchIconButtonLocator = "//span [contains(@class, 'search-icon')]";
+        private const string _searchFormFieldLocator = "//*[@id = 'new_form_search']";
+        private const string _findButtonLocator = "//span [contains (@class, 'bth')]";
+
+
+        [SetUp]
+        public void Setup()
+        {
+            _driver = new ChromeDriver();
+            _action = new Actions(_driver);
+            _driver.Manage().Window.Maximize();
+            _driver.Navigate().GoToUrl("https://www.epam.com/");
+        }
+
+        [Test]
+        public void CheckAvailableCountriesIsPresentOnJobListingsPage()
+        {
+            _driver.FindElement(By.XPath("//*[contains(@class, 'item-link') and @href = '/careers']")).Click();
```

Comment:
Minor comment to make your code more readable. Plase find element first, name it and then do actions.
ex: var searchButton = _driver.FindElement()
searchButton.Click()